features/product/managing_product_association_types/filtering_product_association_types.feature

```diff
@@ -0,0 +1,24 @@
+@managing_product_association_types
+Feature: Filtering product association types
+    In order to filter product association types by name or code
```

Comment:
IMO new feature =~ new file but in this case I agree with you and I will change that